ts/src/profiler.ts

```diff
@@ -126,6 +137,48 @@ async function profileBytes(p: perftools.profiles.IProfile): Promise<string> {
 }
 
 /**
+ * Error which also indicates how long backoff should be in response to the
+ * error.
+ */
+class ResponseError extends Error {
+  backoffMillis?: number;
+  statusCode: number;
+  constructor(response: http.ServerResponse) {
+    let message: number|string = response.statusCode;
```

Comment:
Why it's a number or string? Do we get both from the server?

Reply:
I've updated how this is done.